features/db-check.feature

```diff
@@ -97,3 +97,18 @@ Feature: Check the database
       Access denied
       """
     And STDOUT should be empty
+
+  Scenario: Ensure MySQL defaults are available when as appropriate with --defaults flag
+    Given a WP install
+
+    When I try `wp db check --defaults --debug`
+    Then STDERR should contain:
+      """
+      Debug (db): /usr/bin/env mysqlcheck %s
+      """
+
+    When I try `wp db check --debug`
+    Then STDERR should contain:
+      """
+      Debug (db): /usr/bin/env mysqlcheck --no-defaults %s
+      """
```

Comment:
Also add cases with the `--no-defaults` flag to make sure the extra logic still keeps the `--flag/--no-flag` behavior consistent.

So, `wp db check --no-defaults --debug` should have the same result right now as `wp db check --debug`.

Reply:
Good call. I've added cases for each of the files in which I was also checking for the command.

Reply:
I don't see these added yet, @ChrisWiegman - I was referring to actually using the flag `--no-defaults` in the command that you `When I try ...`, to ensure the flag handling is not broken with regards to negating it with `--no-...`.